ui/app/styles/components/table.scss

```diff
@@ -55,7 +55,22 @@
     }
   }
 
+  .w-1\/2 {
+    width: 50%;
+  }
+  .w-1\/3 {
+    width: 33.333%;
+  }
+  .w-1\/4 {
+    width: 25%;
+  }
+  .w-1\/5 {
+    width: 20%;
+  }
   .w-2\/5 {
     width: 40%;
   }
+  .w-3\/5 {
```

Comment:
i would have liked to rename those classnames, I don't think backslashes are very practical. :\

Reply:
@gregone feel free to change them. I’m not precious :)